maadmin.js

```diff
@@ -51,46 +39,55 @@ app.post('/maadmin/api/createAccount', (req, res) => {
 })
 
 app.post('/maadmin/api/logout', (req, res) => {
-  admin.logout();
+  maadmin.logout(req.body.username);
   res.set('Access-Control-Allow-Origin', '*');
   res.status(200);
   res.json({success: true});
 })
 
-app.get('/maadmin/api/leagueNames', (req, res) => {
-  leagues = admin.getLeagueNames(leagueNames => {
+app.get('/maadmin/api/:user/leagueNames', (req, res) => {
+  var user = maadmin.users[req.params.user];
+
+  user.getLeagueNames(leagueNames => {
     res.set('Access-Control-Allow-Origin', '*');
     res.status(200);
     res.json(leagueNames)
   });
 })
 
-app.get('/maadmin/api/teamList/:league', (req, res) => {
-  admin.leagues[req.params.league].getTeamNames(teams => {
+app.get('/maadmin/api/:user/teamList/:league', (req, res) => {
+  var user = maadmin.users[req.params.user];
+  console.log(req.params.user);
+  
+  user.getLeague(req.params.league).getTeamNames(teams => {
     res.set('Access-Control-Allow-Origin', '*');
     res.status(200);
     res.json(teams);
   });
 })
 
-app.get('/maadmin/api/playerList/:league', (req, res) => {
-  console.log("Getting player names for league: " + req.params.league);
-  admin.leagues[req.params.league].getPlayerNames(teams => {
+app.get('/maadmin/api/:user/playerList/:league', (req, res) => {
+  var user = maadmin.users[req.params.user];
+
+  user.getLeague(req.params.league).getPlayerNames(teams => {
     res.set('Access-Control-Allow-Origin', '*');
     res.status(200);
     res.json(teams);
   });
 })
 
-app.post('/maadmin/api/createLeague', (req, res) => {
-  admin.addLeague(req.body.leaguename, req.body.numDivs, leagueName => {
+app.post('/maadmin/api/:user/createLeague', (req, res) => {
+  //Create a league, body contains leaguename and numdivs (should also include capacity?)
+  var user = maadmin.users[req.params.user];
+
+  user.addLeague(req.body.leaguename, req.body.numDivs, req.body.divCapacity, leagueName => {
     res.set('Access-Control-Allow-Origin', '*');
     res.status(200);
     res.json(leagueName);
   });
 })
 
-app.post('/maadmin/api/addTeam', (req,res) => {
+app.post('/maadmin/api/:user/addTeam', (req,res) => {
   var first = req.body.firstName;
```

Comment:
Let's instead pass in the body of the response, vs parsing through it here. Or we can create an object that contains all this data so we don't have to keep passing in all these params every time